multinet/validation/validate_csv.py

```diff
@@ -0,0 +1,110 @@
+"""Utilities for validating tabular data for upload to Multinet."""
+
```

Comment:
We actually need a bit of a full refactor for where everything goes. I only moved this file during this PR because it was part of a circular dependency that I could most easily break by moving the code. Since that was why I moved the code, I very much prefer not to move other code just for immediate consistency.

I will open an issue to describe the general refactoring task I want to investigate, and I will mention your concern about moving validation code here in that issue as well. But at the moment, we shouldn't take that on since it's outside the scope of this already long-lived branch

Reply:
See #429